v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,683 @@
+                            // We need to have a request ID when creating a code in order to detect possible bots, suspicious IP addresses, etc.
+                            const requestId = (await input.options.req.getJSONBody()).requestId;
+                            if(!requestId) {
+                                // make sure to throw an error or correctly handle this error case
+                                throw new Error("requestId is required");
+                            }
+
+                            // ... your code here ...
+                            let session = await Session.getSession(input.options.req, input.options.res, {
+                                overrideGlobalClaimValidators: () => [],
+                            });
+                            let phoneNumber: string = session.getAccessTokenPayload().phoneNumber;
+
+                            // we check the anomaly detection service before calling the original implementation of signUp
+                            let securityCheckResponse = await handleSecurityChecks({ ...input, requestId, phoneNumber });
+                            // ... security check code here ...
+
+                            return originalImplementation.createCodePOST(input);
+                        },
+                        resendCodePOST: async function (input: any) {
+                            // We need to have a request ID when resending the code in order to detect possible bots, suspicious IP addresses, etc.
+                            const requestId = (await input.options.req.getJSONBody()).requestId;
+                            if(!requestId) {
+                                // make sure to throw an error or correctly handle this error case
+                                throw new Error("requestId is required");
+                            }
+
+                            // ... your code here ...
+                            let session = await Session.getSession(input.options.req, input.options.res, {
+                                overrideGlobalClaimValidators: () => [],
+                            });
+                            let phoneNumber: string = session.getAccessTokenPayload().phoneNumber;
+
+                            // we check the anomaly detection service before calling the original implementation of signUp
+                            let securityCheckResponse = await handleSecurityChecks({ ...input, requestId, phoneNumber });
+                            // ... security check code here ...
+
+                            return originalImplementation.resendCodePOST(input);
+                        },
+                    };
+                },
+            },
+            // highlight-end
+        }),
+        Session.init({ /* ... */ })
+    ]
+})
+```
+</TabItem>
+
+<TabItem value="go">
+go
+</TabItem>
+
+<TabItem value="python">
+python
+</TabItem>
+
+<TabItem value="otherFrameworks">
+other frameworks
+</TabItem>
```

Comment:
add a section about Third party as well, and just say that we do not recommend using the security service for this cause it's the third party provider's auth that would do security checks anyway.

Reply:
done